src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -736,41 +736,98 @@ public String getInstructorFeedbackSessionActions(FeedbackSessionAttributes sess
             "onclick=\"return toggleDeleteFeedbackSessionConfirmation('" + session.courseId + "','" + session.feedbackSessionName + "');\" " +
             disableDeleteSessionStr + ">Delete</a> "
         );
-        result.append(
-            "<a class=\"btn btn-default btn-xs btn-tm-actions session-submit-for-test" + (hasSubmit ? "\"" : DISABLED) +
-            "href=\"" + getInstructorFeedbackSessionSubmitLink(session.courseId,session.feedbackSessionName) + "\" " +
-            "title=\"" + Const.Tooltips.FEEDBACK_SESSION_SUBMIT + "\" data-toggle=\"tooltip\" data-placement=\"top\"" +
-            disableSubmitSessionStr + ">Submit</a> "
-        );
+        
+        if (hasSubmit) {
+            result.append(
+                    "<a class=\"btn btn-default btn-xs btn-tm-actions session-submit-for-test" + "\" " +
+                    "href=\"" + getInstructorFeedbackSessionSubmitLink(session.courseId, session.feedbackSessionName) + "\" " +
+                    "title=\"" + Const.Tooltips.FEEDBACK_SESSION_SUBMIT + "\" data-toggle=\"tooltip\" data-placement=\"top\"" +
+                    disableSubmitSessionStr +">Submit" +
+                    "</a> "
+                );
+        } else {
```

Comment:
I change back to ongoing and work on that and see how many of these branches I can drop